src/jsMain/kotlin/dev.fritz2.kitchensink/demos/formControlDemo.kt

```diff
@@ -342,8 +342,8 @@ fun RenderContext.formControlDemo(): Div {
             +" is passed as metadata value. For the final validation on button click, "
             c("Registration")
             +" is passed. This allows the validator to reflect the current phase and process blank "
-            +" data after the button click only. The exact example code is shown below - its length is mostly"
-            +" due to the added validation content."
+            +" data after the button click only. Due to the length, we only show an sufficient excerpt of the real "
+            +"source."
         }
```

Comment:
To keep it short, the following example shows only an excerpt of the real source code.